avm/res/api-management/service/api-version-set/main.bicep

```diff
@@ -4,10 +4,10 @@ metadata description = 'This module deploys an API Management Service API Versio
 @sys.description('Conditional. The name of the parent API Management service. Required if the template is used in a standalone deployment.')
 param apiManagementServiceName string
 
-@sys.description('Optional. API Version set name.')
-param name string = 'default'
+@sys.description('Required. API Version set name.')
+param name string
```

Comment:
For changes like these these I have to ask: Can the value be anything but `default`?
For example, we have this constraint for `blobServices` in the Storage Account module